src/amg/azext_amg/tests/latest/test_amg_migrate_scenario.py

```diff
@@ -241,9 +241,14 @@ def test_amg_migrate_remapping(self, resource_group):
             self.kwargs.update({
                 'dashboardDefinitionDatasource': test_dashboard_with_datasource,
                 'dashboardTitle4': dashboard_title + '4',
-                'dataSourceDefinitionDifferentUID': test_data_source_different_uid
+                'dataSourceDefinitionDifferentUID': test_data_source_different_uid,
+                'dashboardDefinitionDatasourceShortUid': test_dashboard_with_datasource_short_uid,
+                'dashboardTitle5': dashboard_title + '5',
+                'dataSourceDefinitionLongUid2': test_data_source_long_uid2,
+                'dataSourceDefinitionShortUid2': test_data_source_short_uid2,
             })
             self.kwargs['dashboardDefinitionDatasource']['dashboard']['uid'] = 'mg2OAlTVd'  # control the uid to prevent auto generated uid with possible '-' that breaks the command
+            self.kwargs['dashboardDefinitionDatasourceShortUid']['dashboard']['uid'] = 'mg2OAlTVe'  # control the uid to prevent auto generated uid with possible '-' that breaks the command
```

Comment:
Why do we use the same value for both the regular uid and short uid here?

Reply:
Oh, I changed the last value to be 'e' instead of 'd'. The shortUid is because the datasources inside the dashboard use shortuid. We don't care about the dashboard uid.